parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -837,19 +837,19 @@ syncInternal progress srcConn destConn b = time "syncInternal" do
                       traceM $ "  decls: " ++ show ds
                       traceM $ "  edits: " ++ show es
                     (cs, es, ts, ds) <- lift $ runDB destConn do
-                      cs <- filterM (fmap not . runDB destConn . isCausalHash' . fst) branchDeps
-                      es <- filterM (fmap not . runDB destConn . patchExists') es
-                      ts <- filterM (fmap not . runDB destConn . termExists') ts
-                      ds <- filterM (fmap not . runDB destConn . declExists') ds
+                      cs <- filterM (fmap not . isCausalHash' . fst) branchDeps
+                      es <- filterM (fmap not . patchExists') es
+                      ts <- filterM (fmap not . termExists') ts
+                      ds <- filterM (fmap not . declExists') ds
```

Comment:
These extra `runDB destConn`s are not needed given the outer one (which I suppose is ignored? one of them is ignored.)